app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -30,6 +30,47 @@ Locality-aware load balancing is enabled by default unlike its predecessor [loca
+    - **`key`** - defines tag for which affinity is configured. The tag needs to be configured on the inbound of the service. In case of Kubernetes, pod needs to have an annotations or a label. On Universal user needs to define it on the inbound of the service. In case there no match, the dataplane match to the last group with the lowest weight.
+    - **`weight`** - (optional) weight of the tag used for load balancing. The bigger the weight the higher number of requests is routed to dataplanes with specific tag. By default we will adjust them so that 90% traffic goes to first tag, 9% to next, and 1% to third and so on.
+
+#### CrossZone
+{% warning %}
+Remember that cross-zone traffic requires [mTLS enabled](/docs/{{ page.version}}/policies/mutual-tls).
+{% endwarning %}
+Advanced locality-aware load balancing provides a powerful means of defining how your service should behave when there is no service available in your local zone. With this feature, you have the flexibility to configure the fallback behavior of your service, specifying the order in which it should attempt fallback options and defining different behaviors for instances located in various zones.
+
+- **`crossZone`** - (optional) allows to define behaviour when there is no healthy instances of the service. When not defined, cross zone traffic is disabled.
+  - **`failover`** - defines a list of load balancing rules in order of priority. If a zone is not specified explicitly by name or implicitly using the type `Any`/`AnyExcept` it is excluded from receiving traffic. By default, the last rule is always `None` which means, that there is no traffic to other zones after specified rules.
+    - **`from`** - (optional) defines the list of zones to which the rule applies. If not specified, rule is applied to all zones.
+      - **`zones`** - list of zones.
+    - **`to`** - defines to which zones the traffic should be load balanced.
+      - **`type`** - defines how target zones will be picked from available zones. Available options:
+        - **`Any`** - traffic will be load balanced to every available zone.
+        - **`Only`** - traffic will be load balanced only to zones specified in zones list.
+        - **`AnyExcept`** - traffic will be load balanced to every available zone except these specified in zones list.
+        - **`None`** - traffic will not be load balanced to any zone.
+      - **`zones`** - names of zones
+  - **`failoverThreshold.percentage`** - (optional) defines the percentage of live destination dataplane proxies below which load balancing to the next priority starts. Default: 50%
+
+#### Zone Egress support
+
+Using Zone Egress Proxy in multizone deployment poses certain limitations for this policy. When configuring `MeshLoadbalancingStrategy` with Zone Egress you can only use `Mesh` as a top level targetRef. This is because we don't differentiate requests that come to Zone Egress from different clients, yet. 
```

Comment:
```suggestion
Using Zone Egress Proxy in multizone deployment poses certain limitations for this feature. When configuring `MeshLoadbalancingStrategy` with Zone Egress you can only use `Mesh` as a top level targetRef. This is because we don't differentiate requests that come to Zone Egress from different clients, yet. 
```